webservices_dispatch_action/rerendering.py

```diff
@@ -80,11 +80,22 @@ def comment_and_push_per_changed(
                 "https://conda-forge.org/docs/maintainer/updating_pkgs.html"
                 "#rerendering-with-conda-smithy-locally"
             )
+            global_pinning_url = (
+                "https://github.com/conda-forge/conda-forge-pinning-feedstock/"
+                "blob/master/recipe/conda_build_config.yaml"
+            )
             message = """\
 Hi! This is the friendly automated conda-forge-webservice.
 I tried to rerender for you but ran into some issues, please ping conda-forge/core
 for further assistance. You can also try [re-rendering locally]({}).
-""".format(doc_url)
+
+More information on the issue may be available in the actions tab of the feedstock
+under a "rerender" workflow. The following suggestions might explain the problem:
+
+* Is the `recipe/meta.yaml` file valid?
+* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure
+  that it is compatible with the current [global pinnnings]({}).
```

Comment:
the link to the pinnings is really good. While I might have figured out that `cudnn` was the culprit, I wouldn't have necessarily been able to determine I should use `undefined`... i guess be process of elimination on the selectors?

Reply:
For the specific case in https://github.com/conda-forge/geckodriver-feedstock/pull/17, the value(s) of `cudnn` are also in the pinnings (I think the other values aren't needed as CUDA isn't actually being used but I'm getting out of my depth at this point).

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d8507e01df35678a41ad432e9dd3278f73a689e0/recipe/conda_build_config.yaml#L47-L52